measureme/src/profiler.rs

```diff
@@ -42,11 +42,26 @@ impl<S: SerializationSink> Profiler<S> {
             Arc::new(S::from_path(&paths.string_index_file)?),
         );
 
-        Ok(Profiler {
+        let profiler = Profiler {
             event_sink,
             string_table,
             start_time: Instant::now(),
-        })
+        };
+
+        let mut args = String::new();
+        for arg in std::env::args() {
+            args.push_str(&arg.escape_default().to_string());
+            args.push(' ');
+        }
+
+        profiler.string_table.alloc_metadata(&*format!(
+            "{{ \"start_time\": {}, \"process_id\": {}, \"cmd\": \"{}\" }}",
```

Comment:
This might be a perfect place for using raw string literals.
https://rahul-thakoor.github.io/rust-raw-string-literals/

Reply:
That's a great idea.